tests/main/snap-quota-install/task.yaml

```diff
@@ -29,7 +29,7 @@ execute: |
   echo "Checking that all quotas can be listed"
   snap quotas | cat -n > quotas.txt
   MATCH "     1\s+Quota\s+Parent\s+Constraints\s+Current$" < quotas.txt
-  MATCH "     2\s+group-one\s+memory=400MB\s*$" < quotas.txt
+  MATCH "     2\s+group-one\s+memory=400MB(\s*|\s*memory=[0-9.a-zA-Z]+)\s*$" < quotas.txt
```

Comment:
Please add a comment above this line, explaining that some memory is always being used. But shouldn't we be a bit less tolerant about the last column format? If it's only a couple of pages, I guess we are talking about KB, so we could avoid specifying `a-zA-Z` but rather match the exact sequence.

Reply:
I'd rather not be less tolerant here, as we already have tests that are very exact on this area. What amount of memory is in use is totally unrelated to the test - I'll add a comment tho (nice suggestion) explaining that we expect some memory to be in use